docker/README.md

```diff
@@ -3,6 +3,6 @@ A public docker file that has all the dependencies to build and develop the DS3
 
 To use this docker container, run the following command:
 
-`sudo docker run -it spectralogic/ds3_c_docker`
+`sudo docker run -it spectralogic/ds3_python_docker`
```

Comment:
this should be "spectralogic/ds3_python_docker_test" : https://hub.docker.com/u/spectralogic/dashboard/